api/approval.go

```diff
@@ -37,11 +37,12 @@ type PRRApproval struct {
 	Number string `json:"kepNumber" yaml:"kep-number" validate:"required"`
 
 	// TODO: Need to validate these milestone pointers are not nil
-	Alpha      *PRRMilestone `json:"alpha" yaml:"alpha,omitempty"`
-	Beta       *PRRMilestone `json:"beta" yaml:"beta,omitempty"`
-	Stable     *PRRMilestone `json:"stable" yaml:"stable,omitempty"`
-	Deprecated *PRRMilestone `json:"deprecated" yaml:"deprecated,omitempty"`
-	Removed    *PRRMilestone `json:"removed", yaml:"removed,omitempty"`
+	Alpha              *PRRMilestone `json:"alpha" yaml:"alpha,omitempty"`
+	Beta               *PRRMilestone `json:"beta" yaml:"beta,omitempty"`
+	Stable             *PRRMilestone `json:"stable" yaml:"stable,omitempty"`
+	Deprecated         *PRRMilestone `json:"deprecated" yaml:"deprecated,omitempty"`
+	Removed            *PRRMilestone `json:"removed" yaml:"removed,omitempty"`
+	DeprecatedDisabled *PRRMilestone `json:"deprecatedDisabled" yaml:"deprecated-disabled,omitempty"`
```

Comment:
Adding fields here is currently unused, but `Removed` and `Deprecated` fields have already been added here, and they also not be referenced by any code.